pages/common/python.md

```diff
@@ -29,7 +29,7 @@
 
 - Interactively debug a Python script:
 
-`python -m pdb {{script.py}}`
+`python -m pdb {{path/to/file.py}}`
```

Comment:
```suggestion
`python -m {{pdb}} {{path/to/file.py}}`
```
Could you please also put `pip` in line 28 in `{{}}`? I can't make a suggestion there.